velox/external/date/tz.h

```diff
@@ -284,6 +284,33 @@ ambiguous_local_time::make_msg(local_time<Duration> tp, const local_info& i)
     return os.str();
 }
 
+class invalid_timezone
+    : public std::runtime_error
+{
+public:
+    invalid_timezone(const std::string& tz_name);
```

Comment:
nit: why not just defining the body here inline?

Reply:
Was just following the file's code style. It doesn't likely adopt inline definitions.